proxy/http.go

```diff
@@ -66,8 +66,8 @@ var DefaultHTTPResponseParserConfig = HTTPResponseParserConfig{
 
 // HTTPResponseParserConfig contains the config for a given HttpResponseParser
 type HTTPResponseParserConfig struct {
-	dec encoding.Decoder
-	ef  EntityFormatter
+	Dec encoding.Decoder
```

Comment:
If we're going to expose these fields, maybe we should give them a proper name (aka: `Decoder` and `EntityFormatter`)